protostar/compiler/__init__.py

```diff
@@ -0,0 +1,6 @@
+from .project_compiler import (
+    CompilationException,
+    SourceFileNotFoundException,
+    ProjectCairoPathBuilder,
+    ProjectCompiler,
+)
```

Comment:
Do we need this? It's not a request, but personally I tend to skip writing collective `__init__` modules in non-library Python projects 😃

Reply:
I like to use this approach from the following reasons:
- changes in the module structure don't propagate across the codebase
- making imports nicer
```python
from protostar.compiler import CompilationException, ProjectCompiler
```
- defining module's interface

Reply:
👍🏻